### DISCUSSION TITLE
Lucky you: New Repository Rules 🍀  [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#113573] [Pull Requests]
![Header for the Pull Requests category check-in shows a blue starry background with various planets in the distance. The left has some clouds and an octocat in a Hawaiian shirt holding a GitHub mug. To the right there is a blue and red rocket ship taking off.](https://github.com/community/community/assets/92824518/b7f50bb7-5979-4c6f-92aa-8f3dc5347b99)

This check-in is a pot of gold filled with community shoutouts, new releases, and projects to contribute to you - you’re on the right side of the rainbow 🌈

### ⚡Pull Requests Praise: Thanks to these fine folks for keeping our Community awesome 
* @drok: Put together a [detailed post](https://github.com/orgs/community/discussions/112690) of what they expected when adding two new commits at the end of a PR review and then shared the behavior they experienced. I appreciate how organized this was and all the details needed to help someone solve - which could be you! [Read their discussion](https://github.com/orgs/community/discussions/112690) and see if you can assist them. 
* @wshank: Gave us some [great feedback on the Merge Queue](https://github.com/orgs/community/discussions/111224) in their post! We appreciate you helping us improve ❣️. 

### ℹ️ What’s new: The latest Pull Requests updates
* **Repository Rules (Public Beta)**: Configuring merge queue in your repo rulesets is now available in public beta! [Read the changelog](https://github.blog/changelog/2024-02-27-repository-rules-configure-merge-queue-rule-public-beta/) and [join our discussion](https://github.com/orgs/community/discussions/110908). 

### 📗Rousing Reads: GitHub-produced goodies just for you! 
* **How GitHub uses merge queue to ship hundreds of changes every day**: @willsmythe & @lawrencegripper share how merge queue transformed the way GitHub deploys changes to production at scale, so you can do the same for your organization. [Read on for more](https://github.blog/2024-03-06-how-github-uses-merge-queue-to-ship-hundreds-of-changes-every-day/). 
* **Keeping repository maintainer information accurate**: Discover how keeping repository maintainer information accurate through CODEOWNERS files and automating maintenance with tools like cleanowners fosters efficient collaboration and sustainable software projects in [our blogpost](https://github.blog/2024-03-04-keeping-repository-maintainer-information-accurate/).

### 🦸🏻‍♀️ Celebrate Women’s History Month: Level up your GitHub game and open up a PR to get involved with projects who are using technology to make the world a better place. 
* **Postpartum Wellness App**: This project is an app that helps new parents monitor their well-being during the post-partum stage. [Get involved with the project](https://github.com/BekahHW/postpartum-wellness-app). 
* **Women’s Safety Box**: This is an Android App for Women or any person, which Will Call to an Emergency number, based on a specific voice command like "Help" . It will also send SMS with current GPS coordinates to those numbers. [View the project](https://github.com/ShakilAhmedShaj/Women_Safety_Box). 
* **Tattle Made**: A community of technologists, researchers, and artists working towards a healthier online information ecosystem - visit [their repo](https://github.com/tattle-made) to learn how you can help. 

What projects have you opened PRs on this month? Share them below!

**Comment by Zeus-Aphrodite** [comment]
Thank you for your attention.
